README.md

```diff
@@ -2,28 +2,97 @@
   <img src="https://github.com/pyscf/pyscf-doc/blob/master/logo/pyscf-logo.png" height="80px"/>
 </div>
 
-PySCF documentation
+PySCF Documentation
 ===================
 
-Installation
-------------
+## Overview
 
-* Prerequisites
-    - [sphinx](https://pypi.org/project/Sphinx/)
-    
-    - [sphinxcontrib-bibtex](https://pypi.org/project/sphinxcontrib-bibtex/) v1.0.0
+The PySCF documentation uses `sphinx-doc` and several associated extensions to generate static `html` files for the website.
+To simplify the command line work, we use `make` to manage all building here, see [the Building section](#building) below or just run `make help` for more details.
 
-    - Set `PYTHONPATH` to include the PySCF source directory; otherwise, uncomment `sys.path.append(os.path.abspath('path_to_pyscf'))` in [source/conf.py](source/conf.py).
+## Building the Docs Locally
```

Comment:
Camel case here but not elsewhere?

Reply:
I'll change them to be camel case consistently throughout the `README.md`